pkgs/applications/graphics/graphicsmagick/default.nix

```
@@ -15,20 +34,39 @@ stdenv.mkDerivation rec {
     ./disable-popen.patch
   ];
 
+  outputs = [ "out" "dev" ];
+
+  enableParallelBuilding = true;
+
   configureFlags = [
     "--enable-shared"
     "--with-frozenpaths"
     "--with-quantum-depth=${toString quantumdepth}"
     "--with-gslib=yes"
+    "--with-modules"
+    "--with-perl"
+    "--with-threads"
   ];
 
   buildInputs =
```

```suggestion
  buildInputs = [
```
If we format this, we might as well do this.